src/platforms/common/aux_serial.c

```diff
@@ -87,6 +87,23 @@ static char aux_serial_transmit_buffer[AUX_UART_BUFFER_SIZE];
 #define usart_set_parity(uart, parity)     uart_set_parity(uart, parity)
 #endif
 
+void bmp_usart_set_baudrate(uint32_t usart, uint32_t baud_rate)
+{
+#if defined(USART_CR1_OVER8)
+	const uint32_t clock = rcc_get_usart_clk_freq(usart);
+	/* Autoselect default OVER16 unless a very high baudrate requested */
+	if (baud_rate <= clock / 16U) {
+		usart_set_oversampling(usart, USART_OVERSAMPLING_16);
+	} else if ((clock / 16U < baud_rate) && (baud_rate <= clock / 8U)) {
+		usart_set_oversampling(usart, USART_OVERSAMPLING_8);
+	} else {
```

Comment:
The braces around the sections of this `if` block are unnecessary as are the parens in the `else if` clause, please drop them.

Reply:
This does include the else block too, we forgot to make clear.

Reply:
Removed all braces.